hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/decorators/SPUIButtonStyleNoBorderWithIcon.java

```diff
@@ -25,18 +26,33 @@ public Button decorate(final Button button, final String style, final boolean se
 
         this.button = button;
 
+        button.addStyleName(ValoTheme.LABEL_SMALL);
+        button.addStyleName(ValoTheme.BUTTON_BORDERLESS_COLORED);
+
         setButtonStyle(style, setStyle);
         setButtonIcon(icon);
 
-        button.addStyleName(ValoTheme.LABEL_SMALL);
         button.setSizeFull();
 
+        setButtonStyle(style, setStyle);
```

Comment:
You are calling `setButtonStyle(style, setStyle);` already in line 32, so why once more?